src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -491,7 +495,50 @@ private String beschreibung(String strUrlFeed) {
             } else
                 return seite3.extract("<p class=\"subtitle\">", "<p class=\"teasertext\" itemprop=\"description\">", "<");
         }
+}
+
+	private boolean isAllTextsEmpty(String... aTexts) {
+		Boolean isAllTextsEmpty = null;
+		for (String text : aTexts) {
+			isAllTextsEmpty = (isAllTextsEmpty == null || isAllTextsEmpty) && StringUtils.isEmpty(text);
+		}
+		return isAllTextsEmpty == null || isAllTextsEmpty;
+	}
+
+	/**
+	 * Searches the Seite for a quality auto to get a M3U8 URL. If the URL is
+	 * from WRD it searches for the URLs of the MP4 files.
+	 *
+	 * @param aSeiteStringExtractor
+	 *            The Seite.
+	 * @return A Map containing the URLs and Qualities which was found. An empty
+	 *         Map if nothing was found.
+	 */
+	private Map<Qualities, String> searchForUrlsWithM3U8(final MSStringBuilder aSeiteStringExtractor) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this method into "ThemaLaden". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3398)